athena-query-execution-pipeline/athena_query_execution/configs.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is there a reason we don't leverage `ddk.json` here?

Reply:
I wanted to keep ddk infra config separate from usecase config, The config file can have 100s of queries for each env which could make the ddk.json file too crowded.

Reply:
Understood, fine by me